metron-deployment/ansible/playbooks/sensor_install.yml

```diff
@@ -46,6 +46,19 @@
   tags:
     - sensors
 
+#
+# deploys all components required to simulate and then capture packets
+#
+- hosts: sensors
+  become: true
+  roles:
+    - { role: ambari_gather_facts }
+    - { role: tap_interface }
+    - { role: pcap_replay }
+    - { role: pycapa }
+  tags:
+    - pcap
```

Comment:
Allows us to deploy all of the components required for testing packet capture using the tag 'pcap'.